src/plugins/intel_cpu/src/nodes/mkldnn_fullyconnected_node.cpp

```diff
@@ -162,6 +242,15 @@ void MKLDNNFullyConnectedNode::execute(mkldnn::stream strm) {
                     mkldnn::memory::desc newMemDesc(oldMem.get_desc().reshape(normalizedDims));
                     mkldnn::memory newMem(newMemDesc, oldMem.get_engine(), oldMem.get_data_handle());
                     primArgs.at(argType) = newMem;
+                } else {
```

Comment:
Could you please move the dims normalization part to the `preapareParams()` stage and check  the input tensor rank to determine whether we need pointers update.